src/reason-parser/reason_syntax_util.cppo.ml

```diff
@@ -24,6 +24,9 @@ open Ast_mapper
 open Parsetree
 open Longident
 
+(* Rename labels in function definition/application and records *)
+let rename_labels = ref false
```

Comment:
We're going to flip this to true for non-bs usage pretty soon. Is that going to be easy to coordinate with BS in the next next update? Would it be easier to just flip this to true, and let BS flip it to false from the outside? Don't need to block on that decision, but just curious if there's a default that can be set here that makes it easier later.

Reply:
Sure flipping on the bs side can be done. One other question is what to do longer term with tests, if bs and native behave differently.

Reply:
This is gonna be my last comment in this discussion, since it seems like my concerns aren't really getting heard: I strongly disagree that BuckleScript and Reason should behave differently in the long term. This is introducing a dangerous precedent for deviating even more from OCaml.

I guess I'm OK with this PR to go in right now to fix the immediate issue of the breaking change we accidentally introduced. But I'm very strongly in favor or moving the ecosystem towards a world where we do output an OCaml compatible AST.

Reply:
The longer term situation needs to be looked at carefully, and will take more time.
One of the proposals is to have a compatibility mode, where you don't get to use keywords from the other language at all. In that proposal, bs and native would behave identically.
One evolution of the compatibility mode is to also ensure that platform-specific libraries/features are not used. So Js.* can't be used in code shared between Js and native, and you get an error/warning rather than having to think about it yourself. This is kind of the dual approach to trying to unify libraries: let libraries specialize optimally in their domain when it's important, but document the differences.

Reply:
Let's merge this PR just to unbreak people, but we can continue the discussion on this thread about what to do next.